Sources/Orbit/Components/SegmentedSwitch.swift

```diff
@@ -0,0 +1,157 @@
+import SwiftUI
+
+/// Showing two choices from which only one can be selected.
+///
+/// Currently SegmentedSwitch allows to have 2 segments.
+/// SegmentedSwitch can be in error state only in unselected state.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/segmentedswitch/)
+public struct SegmentedSwitch<Value>: View where Value: LosslessStringConvertible & Equatable {
+
+    @Binding private var selection: Value?
+    let label: String
+    let _message: Message?
```

Comment:
`message`

Reply:
There is another property `private var message: Message? {` as we agreed to hide error message in case there is a selection.

So `let _message` is a value provided by client, `var message` is used in layout